devel/libenkf/tests/enkf_workflow_job_test_version.c

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright (C) 2016  Statoil ASA, Norway.
+
+   The file 'enkf_workflow_job_test_version.c' is part of ERT -
+   Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <ert/util/ert_version.h>
+#include <ert/util/test_util.h>
+#include <ert/enkf/ert_test_context.h>
+
+/*
+  These version functions are overrideed here to get a consistent
+  version mapping.
+*/
+
+int version_get_major_ert_version( ) {
+  return 1;
+}
+
+
+int version_get_minor_ert_version( ) {
+  return 2;
+}
+
+
+const char* version_get_micro_ert_version( ) {
```

Comment:
That is maybe a good idea, but the `micro` name is already in use - also externally, the code you have commented on here is only the *for-testing-override* of the methods.